tests/unit/integrations/watchers/indexable-home-url-option-watcher-test.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Unit\Integrations\Watchers;
+
+use Brain\Monkey;
+use Mockery;
+use Yoast\WP\SEO\Conditionals\Admin_Conditional;
+use Yoast\WP\SEO\Helpers\Indexable_Helper;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast\WP\SEO\Helpers\Post_Type_Helper;
+use Yoast\WP\SEO\Integrations\Watchers\Indexable_HomeUrlOption_Watcher;
+use Yoast\WP\SEO\Presenters\Admin\Indexation_Permalink_Warning_Presenter;
+use Yoast\WP\SEO\Tests\Unit\TestCase;
+
+/**
+ * Class Indexable_HomeUrlOption_Watcher_Test.
+ *
+ * @group indexables
+ * @group integrations
+ * @group watchers
+ *
+ * @coversDefaultClass \Yoast\WP\SEO\Integrations\Watchers\Indexable_HomeUrlOption_Watcher
+ * @covers ::<!public>
+ */
+class Indexable_HomeUrlOption_Watcher_Test extends TestCase {
```

Comment:
See previous comment regarding class name and file name consistency and length.

Reply:
Fixed